Yoakke.Lsp.Generator/Program.cs

```diff
@@ -62,13 +61,13 @@ static void Translate(DefBase def)
         {
             switch (def)
             {
-            case InterfaceDef i: 
-                Translate(i); 
-                break;
-            case NamespaceDef n: 
-                Translate(n); 
-                break;
-            default: throw new InvalidOperationException();
+                case InterfaceDef i:
```

Comment:
I would suggest reverting this indentation. We might need to consider switching to this style as it's more traditional C#. For now, I'd recommend sticking with the projects style for consistency.

Reply:
Hmm... this accidental change is automated by my IDE, will revert